src/pages/kyunghyun/Main/comment/CommentInfo.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component, Fragment } from 'react';
+
+class CommentInfo extends Component {
+  state = {
+    comment: '',
+    editing: false,
+  }; // 수정을 위한 state값
+
+  handleRemove = () => {
+    const { info, onRemove } = this.props;
+    onRemove(info.id);
+  }; // 댓글을 삭제하는 함수 main의 handleRemove를 props로 받아옴(onRemove)
+
+  handleToggleEdit = () => {
+    const { info, onUpdate } = this.props;
+    if (this.state.editing) {
+      onUpdate(info.id, {
+        comment: this.state.comment,
+      }); // 만약에 editing값이 true이면 특정 id를 가진 info의 comment를 현재 state의 comment로 변경
+    } else {
+      this.setState({
+        comment: info.comment,
+      });
+    }
+    this.setState({
+      editing: !this.state.editing,
+    }); // 위 함수들이 실행된 이 후 editing 값을 현재 state editing값의 반대값으로 설정(true이면 False로/ false이면 true로)
+  };
```

Comment:
```suggestion
  handleToggleEdit = () => {
    const { info, onUpdate } = this.props;
    const { editing, comment } = this.state
    
    if (editing) {
      onUpdate(info.id, { comment });
    } else {
      this.setState({ comment: info.comment });
    }
    
    this.setState({ editing: !editing });
  };
```
구조분해 할당 하시고 줄 바꿈만 적당히 하셔도 가독성 좋아질거 같아요